scripts/html5.js

```diff
@@ -735,32 +735,6 @@ H5P.VideoHtml5 = (function ($) {
       });
     });
 
-    // xAPI extension events for video.
-    self.on('seeked', function (event) {
-      this.triggerXAPI('seeked', event.data);
-    });
-    self.on('volumechange', function (event) {
-      this.triggerXAPI('interacted', event.data);
-    });
-    self.on('finished', function (event) {
-      // Triggered as finished to be seperate from H5Ps completed,
-      // but statement is sent as completed and differentiated by object.id
-      this.triggerXAPI('completed', event.data);
-    });
-    self.on('fullscreen', function (event) {
-      // @todo: Not currently used.
```

Comment:
Not sure if we should keep these todo marks; Kirk put them there as hints about what the specific player did not implement. In this case, the html5 player doesn't fire a fullscreen event when it goes fullscreen.